src/filters/log.ts

```diff
@@ -10,6 +11,11 @@ export class Log extends Filter {
 
     async run(message: Message): Promise<boolean> {
         console.log(`${message.author.username}#${message.author.discriminator}: ${serializeMessage(message)}`);
+        if(config.attachmentOnlyChannels.find(channel => channel == message.channel.id) && !message.author.bot && message.attachments.size == 0 && !message.content.includes("http://") && !message.content.includes("https://")){
+            let response = message.reply(`Chatting is not allowed in this channel, your message will be deleted in ${(3 * config.deleteTime)/1000}s. If you want to reply to media, please do so in a thread.`);
+            setTimeout(async () => {  (await response).delete() }, 3 * config.deleteTime);
+            setTimeout(async () => {  message.delete() }, 3 * config.deleteTime);
+        }
```

Comment:
This should be its own `attachment_only.ts` (`AttachmentOnly`) filter, not a part of the `Log` filter.